src/postprocess/apirequests.py

```diff
@@ -14,7 +14,7 @@ def init_api(url, username, password):
     return dhis2api.Dhis2Api(url, username, password)
 
 
-def wait_for_server(api, timeout=900):
+def wait_for_server(api, timeout=43200):
```

Comment:
A timeout of 43200 (12h)?

Reply:
timeout passed to config file ("timeout" : seconds)